src/build_and_deploy.sh

```diff
@@ -0,0 +1,106 @@
+#
+# The `build_and_deploy.sh` file centralizes functions related to kw bd.
+#
+
+include "${KW_LIB_DIR}/deploy.sh"
+include "${KW_LIB_DIR}/build.sh"
+
+# kw bd is a shortcut to build and deploy a new kernel.
+# It also supports arguments related to git bisect usage.
+#
+# The idea of using git bisect along with kw bd is to find the
+# first commit where building and/or deploying the kernel failed.
+#
+# @raw_options String with all user options
+#
+function build_and_deploy_main()
+{
+
+  local raw_options="$*"
+
+  if [[ "$1" =~ -h|--help ]]; then
+    build_and_deploy_help "$1"
+    exit 0
+  fi
+
+  last_commit=$(eval git rev-parse HEAD)
```

Comment:
Why do you need to use `eval` here? Why not use the git command directly?

Reply:
Also, running this command at this stage looks weird to me since the standard `kw bd` will be the main option. I guess it will be way less common for the end user to use the bisect option when compared with the normal build and deploy.